templates/repo/commits.tmpl

```diff
@@ -3,9 +3,9 @@
 	{{template "repo/header" .}}
 	<div class="ui container">
 		{{template "repo/sub_menu" .}}
-		<div class="ui secondary stackable menu mobile--margin-between-items">
-		{{template "repo/branch_dropdown" dict "root" .}}
-			<div class="fitted item">
+		<div class="flex-button-row df ac sb fw mb-4 mt-3">
```

Comment:
At this point this probably needs to have its own classname...

Reply:
It already has, but it's more bytes down the wire if I write these classes in CSS, so I prefer it this way.

Reply:
I think this is fine, I could add another 100 bytes of CSS or keep 10 bytes in HTML, I prefer the latter.

Reply:
I do not like these abbr names. There was a similar discussion.

https://github.com/go-gitea/gitea/pull/17898#issuecomment-986260674

Reply:
We could always replace with more verbose names later (maybe tailwind's).